packer/scripts/almalinux/hpc_image.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+yum install -y git
+
+df -h
+
+mkdir -p /mnt/resource
+chmod 777 /mnt/resource
+cd /mnt/resource
+
+git clone https://github.com/edwardsp/azhpc-images # -b centos-hpc-20220112
```

Comment:
can we avoid this dependency and uses the regular public repo on a fix version and apply a patch ?

Reply:
not easily at the moment - an issue has been created to resolve - #1395